packages/frontend/src/common/components/ArticleBox.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { PropsWithChildren } from "react";
+import { Box, useTheme } from "@mui/material";
+
+export default function ArticleBox({ children }: PropsWithChildren) {
+  const theme = useTheme();
+  return (
+    <Box sx={{ justifyContent: "center", display: "flex", margin: theme.spacing(4) }}>
```

Comment:
We also have a custom implementation of `FlexBox`. You could use that here :)

Also we preferred the property styles instead of sx. Check `FlexBox.tsx` for an example